tasks/add_additional_nrpe_checks.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Download git repos
+  git:
+    repo: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    version: "{{ item.version }}"
+    force: "{{ item.force |default(True) }}"
+    verify_commit: "{{ item.verify_commit | default(False) }}"
+  with_items: "{{ additional_nrpe_checks }}"
+  when: item.type == 'git'
+
+- name: Default nrpe command argument when nagios_dont_blame is true
+  set_fact:
+    default_argument: "$ARG1$"
```

Comment:
Here I had a moments hesitation, should it be single 'quoted' or are there some corner cases where some weird variable expansion can happen. OTOH the way it's currently used looks fine.

Reply:
I think that it is in bash and puppet where we care about single vs. double quotation.